cypress/integration/visual-testing/homepage-test.spec.js

```diff
@@ -0,0 +1,41 @@
+import { baseURL, eyesOpen } from '../../utils';
+describe('Visual  Regression Tests', () => {
+  beforeEach(() => {
+    cy.visit(baseURL);
+    eyesOpen('Home Page');
+  });
+  afterEach(() => {
+    cy.eyesClose();
+  });
+
+  it('should display sign up options', () => {
+    cy.get('#navigation-butt').click();
+    cy.get('#navigation-butt').should('be.visible');
+  });
+
+  it('should redirect to code of conduct', () => {
+    const elem = cy.get('[data-cy-href=codeofconduct]');
+    elem.contains('Code of Conduct');
+    return elem.invoke('attr', 'href').then(href => {
+      cy.visit(`${baseURL}${href}`);
+      cy.wait(2000);
```

Comment:
In general, it is not recommended to use `wait` unless you have a very good reason as it is an anti-pattern.See https://docs.cypress.io/api/commands/wait.html#Wait-for-an-arbitrary-period-of-milliseconds for more information about this.